backend/chalice/api_server/Makefile

```diff
@@ -14,6 +14,7 @@ export PRIVATE_SUBNET_ID_1=$(shell cut -d' ' -f1 <<<"$(PRIVATE_SUBNET_IDS)")
 export PRIVATE_SUBNET_ID_2=$(shell cut -d' ' -f2 <<<"$(PRIVATE_SUBNET_IDS)")
 export PRIVATE_SUBNET_ID_3=$(shell cut -d' ' -f3 <<<"$(PRIVATE_SUBNET_IDS)")
 export SECURITY_GROUP_ID=$(shell aws ec2 describe-security-groups | jq -rc '.SecurityGroups[] | select(.GroupName == "corpora-api-lambda-$(DEPLOYMENT_STAGE)") | .GroupId')
+export DEPLOYMENT_STAGE ?= test
```

Comment:
Does this line (17) make 18-20 obsolete?

While this addition creates guarantees for running tests, it completely sidesteps the error checking 18-20 provides during development/developer error. If this isn't necessary (e.g. possibly added in code?), I would opt to keep the error handling intact. Thanks!

Reply:
I'll move it to .github/workflows/push-test.yml